types/cobe/cobe-tests.ts

```diff
@@ -0,0 +1,30 @@
+import createGlobe from 'cobe';
```

Comment:
Since this package is declared as `commonjs`, I believe that this needs to be a `require`.